examples/Gear.jcad

```diff
@@ -978,7 +950,7 @@
n0\n\n0101000\n+126 0 -124 0 *\nVe\n1e-07\n5.21962600000001 -9.687094 0\n0 0\n\n0101101\n*\nEd\n 1e-07 1 1 0\n1  58 0 0 0.454346057207502\n2  913 292 5 0 0.454346057207502\n2  914 292 7 0 0.454346057207502\n2  915 282 8 0 0.454346057207502\n2  916 3 15 0 0.454346057207502\n2  917 293 19 0 0.454346057207502\n2  918 293 21 0 0.454346057207502\n2  919 3 22 0 0.454346057207502\n2  920 294 26 0 0.454346057207502\n2  921 294 28 0 0.454346057207502\n2  922 3 29 0 0.454346057207502\n2  923 295 33 0 0.454346057207502\n2  924 295 35 0 0.454346057207502\n2  925 3 36 0 0.454346057207502\n2  926 296 40 0 0.454346057207502\n2  927 296 42 0 0.454346057207502\n2  928 3 43 0 0.454346057207502\n0\n\n0101000\n+124 0 -122 0 *\nVe\n1e-07\n5.33005600000002 -10.002354 0\n0 0\n\n0101101\n*\nEd\n 1e-07 1 1 0\n1  59 0 0 0.334041393393099\n2  929 297 5 0 0.334041393393099\n2  930 297 7 0 0.334041393393099\n2  931 282 8 0 0.334041393393099\n2  932 3 15 0 0.334041393393099\n2  933 298 19 0 0.334041393393099\n2  934 298 21 0 0.334041393393099\n2  935 3 22 0 0.334041393393099\n2  936 299 26 0 0.334041393393099\n2  937 299 28 0 0.334041393393099\n2  938 3 29 0 0.334041393393099\n2  939 300 33 0 0.334041393393099\n2  940 300 35 0 0.334041393393099\n2  941 3 36 0 0.334041393393099\n2  942 301 40 0 0.334041393393099\n2  943 301 42 0 0.334041393393099\n2  944 3 43 0 0.334041393393099\n0\n\n0101000\n+122 0 -120 0 *\nVe\n1e-07\n4.94286600000001 -10.235254 0\n0 0\n\n0101101\n*\nEd\n 1e-07 1 1 0\n1  60 0 0 0.451839026756215\n2  945 302 5 0 0.451839026756215\n2  946 302 7 0 0.451839026756215\n2  947 282 8 0 0.451839026756215\n2  948 3 15 0 0.451839026756215\n2  949 303 19 0 0.451839026756215\n2  950 303 21 0 0.451839026756215\n2  951 3 22 0 0.451839026756215\n2  952 304 26 0 0.451839026756215\n2  953 304 28 0 0.451839026756215\n2  954 3 29 0 0.451839026756215\n2  955 305 33 0 0.451839026756215\n2  956 305 35 0 0.451839026756215\n2  957 3 36 0 0.451839026756215\n2  958 306 40 0 0.451839026756215\n2  959 306 42 0 0.451839026756215\n2  960 3 43 0 0.451839026756215\n0\n\n0101000\n+120 0 -118 0 *\nVe\n1e-07\n4.502456 -10.389524 0\n0 0\n\n0101101\n*\nEd\n 1e-07 1 1 0\n1  61 0 0 0.466647834024772\n2  961 307 5 0 0.466647834024772\n2  962 307 7 0 0.466647834024772\n2  963 282 8 0 0.466647834024772\n2  964 3 15 0 0.466647834024772\n2  965 308 19 0 0.466647834024772\n2  966 308 21 0 0.466647834024772\n2  967 3 22 0 0.466647834024772\n2  968 309 26 0 0.466647834024772\n2  969 309 28 0 0.466647834024772\n2  970 3 29 0 0.466647834024772\n2  971 310 33 0 0.466647834024772\n2  972 310 35 0 0.466647834024772\n2  973 3 36 0 0.466647834024772\n2  974 311 40 0 0.466647834024772\n2  975 311 42 0 0.466647834024772\n2  976 3 43 0 0.466647834024772\n0\n\n0101000\n+118 0 -116 0 *\nVe\n1e-07\n3.985266 -8.913094 0\n0 0\n\n0101101\n*\nEd\n 1e-07 1 1 0\n1  62 0 0 1.56439478425364\n2  977 312 5 0 1.56439478425364\n2  978 312 7 0 1.56439478425364\n2  979 282 8 0 1.56439478425364\n2  980 3 15 0 1.56439478425364\n2  9
```

Comment:
Shouldn't you remove this property entirely?

Reply:
I mean not leaving `"Shape": ""` in the file

Reply:
When i did that the last time, it made objects invisible.